ref_commits.go

```diff
@@ -40,49 +41,81 @@ func (refCommitsTable) String() string {
 	return printTable(RefCommitsTableName, RefCommitsSchema)
 }
 
-func (refCommitsTable) Resolved() bool { return true }
-
 func (refCommitsTable) Name() string { return RefCommitsTableName }
 
 func (refCommitsTable) Schema() sql.Schema { return RefCommitsSchema }
 
-func (t *refCommitsTable) TransformUp(f sql.TransformNodeFunc) (sql.Node, error) {
-	return f(t)
+func (t *refCommitsTable) WithFilters(filters []sql.Expression) sql.Table {
+	nt := *t
+	nt.filters = filters
+	return &nt
 }
 
-func (t *refCommitsTable) TransformExpressionsUp(f sql.TransformExprFunc) (sql.Node, error) {
-	return t, nil
+func (t *refCommitsTable) WithProjection(colNames []string) sql.Table {
+	return t
```

Comment:
doesn't accept this table pushdowns of projections?

Reply:
same answer